content/articles/how-to-ground-truth-label-your-datasets-for-creating-autonomous-driving-systems-using-matlab-ground-truth-app/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-ground-truth-label-your-datasets-for-creating-autonomous-driving-systems-using-matlab-ground-truth-app/
+title: How to ground truth label your datasets for creating autonomous driving systems using matlab ground truth app
+description:
+author: linet-achieng
+date: 2021-11-26T00:00:00-12:15
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/curve-fitting-using-matlab/hero.jpg
+    alt: How to ground truth label your datasets for creating autonomous driving systems using matlab ground truth app Hero Image
+---
+Ground truth labeling is the process of checking the results of machine learning for accuracy in the real world. For example, autonomous driving systems are cars that can move without a human driver or human intervention. 
+<!--more-->
+Instead, these types of cars perceive the environment using sensors. The ground truth labeler app allows us to label videos and images for automotive applications.
+
+We all know that labeling is a tedious and time-consuming process but necessary to create test data to evaluate our perception algorithms. A ground truth labeler is a tool that we have created to reduce some of these pinpoints. 
+
+In this tutorial, we will look at creating ground truth data for object detection and semantic segmentation. We will use algorithms to automate the labeling process and integrate ground truth information from other sensors.
+
+### Prerequisites
+To follow along with this tutorial, you'll need to have:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- An understanding of [MATLAB](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+### How to locate the app
+To locate this app:
+- At the top of the window, click on App's and click on the dropdown arrow.
+
+![image](/engineering-education/how-to-ground-truth-label-your-datasets-for-creating-autonomous-driving-systems-using-matlab-ground-truth-app
```

Comment:
the images location seems to be incorrect. Can you check all the images locations and syntax used to refer to the images. Thank you

Reply:
> 

@linetachieng, Kindly check rectify this.

Reply:
@lalith1403, Kindly clarify the comment you made above.